frontend/src/metabase/dashboard/types/display-options.ts

```diff
@@ -1,7 +1,8 @@
 import type { EmbedDisplayControls } from "metabase/dashboard/types/embed-display-options";
 
+export type DashboardFullscreen = boolean;
```

Comment:
nit:

1. I don't think this is used anywhere outside this file so it shouldn't be exported
2. It begs the question of why this needs to have its own type. Aren't all `isFlag` obvious enough that they're boolean?

Reply:
Very true. I honestly thought I had reverted this because this is a bit useless